src/components/fields/DescriptionField.js

```diff
@@ -3,7 +3,7 @@ import React, {PropTypes} from "react";
 function DescriptionField(props) {
   const {id, description} = props;
   if (!description) {
-    return null;
+    return <div />;
```

Comment:
Could you please add a comment before each empty div returned referencing this PR, so the reader can know about why we're doing this? Something like `// See #312: Ensure compatibility with old version of React.`